src/subdomains/generic/kyc/services/kyc.service.ts

```diff
@@ -122,6 +129,41 @@ export class KycService {
     }
   }
 
+  // TODO: Remove temporary cron job
+  @Cron(CronExpression.EVERY_MINUTE)
+  async storeExistingKycFiles(): Promise<void> {
+    try {
+      this.logger.info('Starting Cron Job to store existing KYC files');
+
+      const allUserData = await this.userDataService.getAllUserData();
+
+      for (const userData of allUserData) {
+        if (this.kycFileService.userIsSynced(userData.id)) continue;
```

Comment:
Maybe filter in SQL?

- Add backward navigation property to user data (`kycFiles`)
- Filter using `userDataRepo.findBy({ kycFiles: { id: IsNull() }})`